src/components/LandingPage/LandingPage.jsx

```diff
@@ -1,68 +1,88 @@
+// First welcome page for new users, (if logged in, will reroute user to /discover)
+// reached at path /welcome
+
 import React, { useState } from 'react';
 import { useHistory } from 'react-router-dom';
-import './LandingPage.css';
 
-// CUSTOM COMPONENTS
-import RegisterForm from '../RegisterForm/RegisterForm';
+import Image from '../Images/contriveLanding.png';
+
+
+// MATERIAL UI
+import { makeStyles } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import { spacing } from '@material-ui/system';
+
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    '& > *': {
+      margin: theme.spacing(1),
+    },
+  },
+  paperContainer: {
+    height: '625px',
+    // width:'100%',
+    // maxHeight: '100%',
+    // maxWidth: '100%',
+    backgroundImage: `url(${Image})`,
```

Comment:
This page is looking slick. But that background image is not fully scaled, depending on what monitor I'm on:

**On my macbook**
<img width="500" alt="Screenshot_2021-04-15-13 35 21" src="https://user-images.githubusercontent.com/1153371/114920891-8c359b00-9def-11eb-8a24-6667169a18c3.png">

**On my external monitor**
<img width="500" alt="Screenshot_2021-04-15-13 35 21" src="https://user-images.githubusercontent.com/1153371/114920958-9fe10180-9def-11eb-8f6a-0d6965b5814e.png">

Reply:
Yup. I noticed that too. I think it's in our bug list.